README.md

```diff
@@ -1,6 +1,55 @@
 audiosystem-passthrough
 =======================
 
+**For new adaptations audioflingerglue (and pulseaudio-modules-droid-glue)
```

Comment:
Please write starting which Android base version is considered a new adaptation (if applicable).